### PR TITLE
Fixed bug related to deleting a dataframe in the Data View window

### LINKED ISSUE #7400: Another small bug when trying to delete a dataframe from Data View window
This bug occurs when trying to delete a data frame from the Data View Window. You can try with any dataset.
Right click on the dataframe name from the Data View window as shown in screenshot below, and then click Delete
image
Then you get this error
image
I suspect that this bug is introduced with the new change made in the `ucrSelector`

### PR DESCRIPTION
**Pull request by Patowhiz**
Fixes #7400

@africanmathsinitiative/developers  @N-thony  this is ready for review. Thanks.

### CODE REVIEW

**Comment by lloyddewit** [comment]
@rdstern If you test (and @N-thony reviews), then I think we can merge this quite quickly, thanks

**Review by N-thony — APPROVED** [review]

**Comment by N-thony** [comment]
@lloyddewit  over to you.

File: instat/ucrSelector.vb [hunk 1]
```diff
@@ -153,10 +152,13 @@ Public Class ucrSelector
 
     End Sub
 
-    Private Function GetSelectorFillCondition(iCurrentElementNum As Integer, strElementType As String, lstCombinedMetadataLists As List(Of List(Of KeyValuePair(Of String, String()))), strHeading As String, strDataFrameName As String, arrStrExcludedItems As String(), strDatabaseQuery As String, strNcFilePath As String)
-        Dim strSelectorFillCondition As String
+    Private Function GetSelectorFillCondition(dataFrame As clsDataFrame, strElementType As String, lstCombinedMetadataLists As List(Of List(Of KeyValuePair(Of String, String()))), strHeading As String, arrStrExcludedItems As String(), strDatabaseQuery As String, strNcFilePath As String)
```
Comment (lloyddewit) [inline instat/ucrSelector.vb:155]:
```suggestion
    Private Function GetSelectorFillCondition(dataFrame As clsDataFrame, strElementType As String,
            lstCombinedMetadataLists As List(Of List(Of KeyValuePair(Of String, String()))),
            strHeading As String, arrStrExcludedItems As String(), strDatabaseQuery As String,
            strNcFilePath As String)
```

**Review by lloyddewit — CHANGES_REQUESTED** [review]

**Comment by lloyddewit** [comment]
@rdstern if you test/approve, then we can merge, thanks

**Review by rdstern — APPROVED** [review]
Great.  Seems fine now

**Comment by lloyddewit** [comment]
@Patowhiz The PR title should make sense for a typical RInstat user. I changed the PR title, please check and update if needed. Thanks

**Pull request merged**